<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="ALOHA08-PC" name="com.popflow.Automation.testcases.CreateWorkspaceActivityTest" tests="1" failures="1" timestamp="28 Jul 2020 12:23:11 GMT" time="67.851" errors="0">
  <testcase name="verifyCreateContactActivity" time="67.851" classname="com.popflow.Automation.testcases.CreateWorkspaceActivityTest">
    <failure type="java.lang.AssertionError" message="FAIL :Unable to create contact using create workspace activity">
      <![CDATA[java.lang.AssertionError: FAIL :Unable to create contact using create workspace activity
at org.testng.Assert.fail(Assert.java:96)
at com.popflow.Automation.helperClass.CommonActions.verifyAsserFailWithException(CommonActions.java:303)
at com.popflow.Automation.helperClass.CommonActions.verifyAssertTrue(CommonActions.java:320)
at com.popflow.Automation.pageObjects.CreateWorkspaceActivityPage.verifyCreateContact(CreateWorkspaceActivityPage.java:169)
at com.popflow.Automation.testcases.CreateWorkspaceActivityTest.verifyCreateContactActivity(CreateWorkspaceActivityTest.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyCreateContactActivity -->
</testsuite> <!-- com.popflow.Automation.testcases.CreateWorkspaceActivityTest -->
